### ISSUE TITLE
Other Quality Assessment algorithms

### ISSUE THREAD

**Issue by DonaldTsang** [Rolinh/VQMT#12]
- [ ] https://github.com/aizvorski/video-quality/issues/8
    - [ ] VIF (sometimes called VIF-P or VIFP), Visual Information Fidelity
    - [ ] RECO, Relative Polar Edge Coherence
    - [ ] NIQE, Natural Image Quality Evaluator
    - [ ] 3SSIM, 3-Component Structural Similarity Metric
    - [ ] VQUAD-HD
    - [ ] VQM
    - [ ] UIQ, Universal Image Quality
    - [ ] MSSIM, Motion SSIM
    - [ ] VMAF

**Comment by rolinh** [comment]
Is this a request to implement all these video quality metrics?
I will welcome pull requests but I have no plans to implement them.

**Comment by DonaldTsang** [comment]
@Rolinh this is mainly a rallying declaration to get people started on adding more and consolidating repos.
https://github.com/aizvorski/video-quality/issues/8

**Comment by stoyanovgeorge** [comment]
I also agree with @DonaldTsang that adding those measurements to VQMT will be great. I think we lack a unified tool to evaluate different video quality metrics.
